docs/notebooks/MetaOT.ipynb

```diff
@@ -0,0 +1,760 @@
+{
```

Comment:
<p>Please add a cell below that checks whether we're in colab, as in <a href="https://ott-jax.readthedocs.io/en/latest/notebooks/Sinkhorn_Barycenters.html" target="_blank">here</a> that includes <code>ott-jax</code> and <code>torchvision</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>